l2-contracts/src/UnifiRewardsDistributor.sol

```diff
@@ -86,6 +86,9 @@ contract UnifiRewardsDistributor is IUnifiRewardsDistributor, Ownable2Step, EIP7
      */
     function setNewMerkleRoot(bytes32 newMerkleRoot) external onlyOwner {
```

Comment:
Maybe we shoudl switch to `restricted`. It gives us more flexibility.

Puffer Backend can post new merkle root (role merkleRootPoster), but the `cancelPendingMerkleRoot` could be granted to n watchers. That way, we don't need to deploy another contract (owner) that will have that logic.